### DISCUSSION TITLE
Raspberry Kompatibilität

### DISCUSSION THREAD

**Discussion by To-Fu112** [camueller/SmartApplianceEnabler#372] [Installation und Betrieb]
Mir stellt sich eine Frage:
Habe einen Raspberry (2011.12) hier liegen. Habe mich peinlichst genau an die Installationsanleitung gehalten. Bekomme es aber nicht zum Laufen. Daher erst einmal die allgemeine Frage, ob mein alter PI überhaupt unterstütz wird.
Aufgefallen ist nämlich, dass die rote LED nach der Instattation von Linux nicht erlischt. Kenne mich mit den Geräten aber auch nicht so gut aus.
Vielen Dank im Voraus!

**Comment by camueller** [comment]
Das kann eigentlich nur ein Raspberry Pi 1 sein. Der ist nicht geeignet wie auch in der [Dokumentation](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Raspberry_DE.md) zu lesen ist.

**Reply by To-Fu112** [reply]
Vielen Dank für die schnelle Antwort. War mir nicht sicher, um welches Modell es sich genau handelt. 
Hatte das aus der Dokumentation aber auch überlesen.
